### PR TITLE
[Merged by Bors] - chore: forward-port leanprover-community/mathlib#18880

### PR DESCRIPTION
**Pull request by eric-wieser**
---

* [`ring_theory.finiteness`@`e95e4f92c8f8da3c7f693c3ec948bcf9b6683f51`..`fa78268d4d77cb2b2fbc89f0527e2e7807763780`](https://leanprover-community.github.io/mathlib-port-status/file/ring_theory/finiteness?range=e95e4f92c8f8da3c7f693c3ec948bcf9b6683f51..fa78268d4d77cb2b2fbc89f0527e2e7807763780)

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Finiteness.lean [hunk 1]
```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johan Commelin
 
 ! This file was ported from Lean 3 source module ring_theory.finiteness
-! leanprover-community/mathlib commit e95e4f92c8f8da3c7f693c3ec948bcf9b6683f51
+! leanprover-community/mathlib commit fa78268d4d77cb2b2fbc89f0527e2e7807763780
```
Comment (eric-wieser) [inline Mathlib/RingTheory/Finiteness.lean:7]: * [`ring_theory.finiteness`@`e95e4f92c8f8da3c7f693c3ec948bcf9b6683f51`..`c813ed7de0f5115f956239124e9b30f3a621966f`](https://leanprover-community.github.io/mathlib-port-status/file/ring_theory/finiteness?range=e95e4f92c8f8da3c7f693c3ec948bcf9b6683f51..c813ed7de0f5115f956239124e9b30f3a621966f)

Comment (YaelDillies) [reply Mathlib/RingTheory/Finiteness.lean:7]: That's the wrong link.

Comment (eric-wieser) [reply Mathlib/RingTheory/Finiteness.lean:7]: This is why I don't think we should trust humans to post these links!

Comment (Parcly-Taxel) [reply Mathlib/RingTheory/Finiteness.lean:7]: * [`ring_theory.finiteness`@`e95e4f92c8f8da3c7f693c3ec948bcf9b6683f51`..`fa78268d4d77cb2b2fbc89f0527e2e7807763780`](https://leanprover-community.github.io/mathlib-port-status/file/ring_theory/finiteness?range=e95e4f92c8f8da3c7f693c3ec948bcf9b6683f51..fa78268d4d77cb2b2fbc89f0527e2e7807763780)

**Review by Parcly-Taxel — APPROVED** [review]

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**